includes/wc-template-functions.php

```diff
@@ -2456,7 +2456,7 @@ function wc_display_item_meta( $item, $args = array() ) {
 		) );
 
 		foreach ( $item->get_formatted_meta_data() as $meta_id => $meta ) {
-			$value = $args['autop'] ? wp_kses_post( $meta->display_value ) : wp_kses_post( make_clickable( trim( strip_tags( $meta->display_value ) ) ) );
+			$value = $args['autop'] ? wp_kses_post( $meta->display_value ) : wp_kses_post( make_clickable( trim( $meta->display_value ) ) );
```

Comment:
Well could only remove p and br tags using:

```php
wp_kses_post( trim( preg_replace( '/(<p>|<\/p>|<br*.\/>)/', '', $meta->display_value ) ) )
```
 Or include an extra option to `get_formatted_meta_data()` not apply `wpautop()`.